app/javascript/components/public_health/PatientsTable.js

```diff
@@ -54,6 +54,7 @@ class PatientsTable extends React.Component {
           { field: 'dob', label: 'Date of Birth', isSortable: true, tooltip: null, filter: formatDate },
           { field: 'end_of_monitoring', label: 'End of Monitoring', isSortable: true, tooltip: null, filter: this.formatEndOfMonitoring },
           { field: 'extended_isolation', label: 'Extended Isolation To', isSortable: true, tooltip: 'extendedIsolation', filter: formatDate },
+          { field: 'first_positive_lab_at', label: 'First Positive Lab', isSortable: true, filter: formatDate },
           { field: 'symptom_onset', label: 'Symptom Onset', isSortable: true, tooltip: null, filter: formatDate },
```

Comment:
The Symptom Onset column is blank for asymptomatic cases. It seem from the mock-ups it's supposed to be either "No History" (slide 9) or "None Reported" (slide 15).

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/747/commits/2aeb65106e1843a9fed9dab750620c3d5eb1af0a?file-filters%5B%5D=.js&file-filters%5B%5D=.rb#diff-605db0c430cb0318fcd9986ece737d10d16c2ab6807a7c35737484322530bb49R488